testem.json

```diff
@@ -3,8 +3,15 @@
   "cwd": ".dist-test",
   "test_page": "index.html?hidepassed",
   "disable_watching": true,
+  "launchers": {
+    "NodeQUnit": {
+      "command": "node scripts/util/test-in-node.js",
```

Comment:
it depends on `test` because it needs the `.dist-test` directory to have been built